lesson08/assignment/test-project/src/views/img-page.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="img-page">
+    <main class="main">
+      <section>
+        <h2>Hyrule's Champions and the Elements</h2>
+        <h3> based purely on what my heart tells me but this content is just so obvious to me</h3>
+        <p class="lead p-intro"> content WIP </p>
+      </section>
+
+      <section v-if="champions">
+        <ChampionPage
```

Comment:
Try adding an index as your key when you use v-for loops.  It doesn't break in this context, but it might cause bugs if you add additional functionality to the elements in the loop.  Like a button to hide or delete won't work on the item you would expect.

v-for="(champion, index) in champions"
:key="index"